alf/trainers/policy_trainer.py

```diff
@@ -258,11 +260,13 @@ def _train(self):
         policy_state = self._driver.get_initial_policy_state()
         for iter_num in range(self._num_iterations):
             t0 = time.time()
-            time_step, policy_state = self.train_iter(
+            time_step, policy_state, steps = self.train_iter(
                 iter_num=iter_num,
                 policy_state=policy_state,
                 time_step=time_step)
-            logging.info('%s time=%.3f' % (iter_num, time.time() - t0))
+            logging.info(
```

Comment:
`Metrics/EnvironmentSteps`  shows the throughput ?

Reply:
yeah, but to get the actual steps per second, we always have to do the math and the elapsed time can be read imprecisely.

Reply:
Also what is shown here is the training steps per second, which is different from Metrics/EnvironmentSteps for off-policy methods.